xml/depl_multiple_networks.xml

```diff
@@ -0,0 +1,433 @@
+</screen>
+</step>
+
+<step>
+    <para>
+        Check the configuration of the bridge:
+    </para>
+    <screen>
+sudo ovs-vsctl show 
+</screen>
+</step>
+    </procedure>
+</sect1>   
+    
+<sect1 xml:id="sec.confignet">
+    <title>Configure the Second External Network</title>
+        <para>
+        The following steps add the networks settings, including IP address 
+pools, gateway, routing, and virtual switches. 
+</para>
+
+    <procedure xml:id="pro.confignet">
+        <step>
+            <para>
+First, stop your &chef; client, 
+then add these lines to <filename>/etc/neutron/l3_agent.ini</filename> with no 
+options:
+    </para> 
+    <screen>
+gateway_external_network_id =
+external_network_bridge =       
+    </screen>
+</step>
+
+<step>
+    <para>
+        The following steps add the networks settings, including IP address 
+pools, gateway, routing, and virtual switches.  First, edit 
+<filename>/etc/neutron/plugins/openvswitch/ovs_neutron_plugin.ini</filename> 
+to include the following line:
+</para>
+<screen>
+bridge_mappings = floating:br-public,public2:br-ext1
+</screen>
+    <para>
+        This change tells the l3 agent that it must rely on the
+physnet-to-bridge mappings.
+    </para>
+</step>
+
+<step>
+    <para>
+    Restart <command>neutron-l3-agent</command>and 
+<command>neutron-openvswitch-agent</command>.
+</para>
+</step>
+
+<step xml:id="step.iface_maps">
+    <para>
+        Run either of the following two commands to set up interface mapping. 
+The first example is for Open vSwitch, and the second example is for 
+Linuxbridge users. These examples use another VLAN ID for the second 
+external network, so the interface mapping for Linuxbridge uses the same 
+network interface (eth0). It is enough to map the new network specifying the 
+correct segmentation ID.
+</para>
+    <screen>
+sudo neutron net-create public2 --provider:network_type flat \
+ --provider:physical_network public2 --router:external=True
+
+sudo neutron net-create public2 --router:external True \
+ --provider:physical_network physnet1 --provider:network_type vlan \
+ --provider:segmentation_id 600
+    </screen>
+</step>
+
+<step>
+    <para>
+        If a different network is used then &crow; will create a new interface 
+mapping. A flat network can be used then:
+</para>
+<screen>
+sudo neutron net-create public2 --provider:network_type flat \
+ --provider:physical_network public2 --router:external=True
+</screen>
+</step>
+
+<step>
+    <para>
+   Now create a subnet:     
+</para>
+<screen>
+sudo neutron subnet-create --name public2 --allocation-pool \
+ start=192.168.135.2,end=192.168.135.127 --gateway 192.168.135.1 public2 \
+ 192.168.135.0/24 --enable_dhcp False
+</screen>
+</step>
+
+<step>
+    <para>
+  The next step is to create a second router:  
+</para>
+<screen>
+sudo neutron router-create router2
+</screen>
+</step>
+
+<step>
+    <para>
+ Connect router2 to the second external network:       
```

Comment:
(If you are wondering about what the ellipsis might be replaced with, TDG is always helpful: http://tdg.docbook.org/tdg/5.1/systemitem.html )